### ISSUE TITLE
Doesn't seem to work with a german takeout

### ISSUE THREAD

**Issue by carstenhag** [dmengelt/google-pay-heatmap#1]
Geting the following javascript error after dragging the site into the page.

```
Uncaught TypeError: Cannot read property 'trim' of undefined
    at FileReader.reader.onload ((index):220)
```

It points to `const longitude = location[1].trim();`.

I'll try to take apart the file later to send you an anonymized version.

**Comment by dmengelt** [comment]
hi @carstenhag 

cool thing. I will have a look then.

**Comment by dmengelt** [comment]
@carstenhag where you able to anonymize your file?

**Comment by carstenhag** [comment]
@domi55 Hey, here's the unedited file (had to change the file extension). I did not anonymize it, meh.
[removed]

**Comment by dmengelt** [comment]
Thanks. I downloaded it and immediately removed the link from your comment. Will have a look...

**Comment by dmengelt** [comment]
Fixed with 937726853bd297fdba1d91be76939e73f637954f
